src/main/java/racingcarwinner/Car.java

```diff
@@ -0,0 +1,41 @@
+package racingcarwinner;
+
+public class Car {
+
+    private Position position;
+    private final Name name;
+    public Car(String name){
+        this.position = new Position(0);
+        this.name = new Name(name);
+
+    }
+    public Position getPosition() {
+        return position;
+    }
+
+    public Name getName(){
+        return this.name;
```

Comment:
```suggestion
        return name;
```
다른 부분 처럼 여기서도 this는 생략해주세요!